osu.Game/Screens/OsuGameScreen.cs

```diff
@@ -22,6 +22,8 @@ public abstract class OsuScreen : Screen
 
         internal virtual bool ShowOverlays => true;
 
+        internal virtual bool UseSettingsOverlays => true;

```

Comment:
why introduce a special case for settings? is there a reason for not making it part of `ShowOverlays` above?